engine/table/src/main/java/io/deephaven/engine/table/impl/select/python/ArgumentsChunked.java

```diff
@@ -64,6 +65,9 @@ private void prepareOneChunkedArg(Object[] chunkedArgs, Class<?>[] chunkedArgTyp
             if (argValue != null) {
                 Arrays.fill((double[]) chunkedArgs[argIdx], (double) argValue);
             }
+        } else if (argType == PyObject.class) {
```

Comment:
What does this accomplish?

Reply:
When the Python function returns a non-primitive object, JPY needs to wrap it as PyObject.  Thus the pre-constructed return array must be of this type instead of the generic Java Object, otherwise we'll get a casting exception between a specific Python object and the Java Object.